### PR TITLE
Updates API docs links to github pages.

### PR DESCRIPTION
**Pull request by matthewkmayer**
Fixes https://github.com/rusoto/rusoto.github.io/issues/27 .

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<span style="float:right">[![github](/img/github.svg)](https://github.com/rusoto/rusoto) [![rustdoc](/img/rustdoc.svg)]({{ book.domain }}/rusoto/rusoto/) [![Latest Version](https://img.shields.io/crates/v/rusoto.svg?style=social)](https://crates.io/crates/rusoto)</span>
+<span style="float:right">[![github](/img/github.svg)](https://github.com/rusoto/rusoto) [![rustdoc](/img/rustdoc.svg)](https://rusoto.github.io/rusoto/rusoto/) [![Latest Version](https://img.shields.io/crates/v/rusoto.svg?style=social)](https://crates.io/crates/rusoto)</span>
```
Comment (indiv0) [inline src/README.md:1]: This fix points it to the SSL domain, which is incorrect.

File: src/usage-and-example.md [hunk 2]
```diff
@@ -43,5 +43,5 @@ fn main() {
 }
 ```
 
-[api-documentation]: {{ book.domain }}/rusoto/rusoto/
+[api-documentation]: https://rusoto.github.io/rusoto/rusoto/
```
Comment (indiv0) [inline src/usage-and-example.md:46]: Same issue here.

**Comment by matthewkmayer** [comment]
I'm not following why https for the github pages link is incorrect.  Can you elaborate?  Works on my machine and all. 😄

**Comment by indiv0** [comment]
That's weird... on my machine I get this as a response:

```xml
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>25E97D09986E8AAE</RequestId><HostId>/3I3Ku8HGLlzm+wg7SpUtyqRRYL8/+nIjYmsyc1ynGJIHc60uUwrfmlWG/KC2bJko0kCNhdiVhw=</HostId></Error>
```

**Comment by matthewkmayer** [comment]
I'm going to merge this since I've checked with others and they are not seeing the same issue as @indiv0 .  I have confirmed others see the issue this PR fixes, so I'm rolling on with it.  

We'll open a new ticket if this breaks things.

Changes will be available in ~24 hours.

**Pull request merged**